roles/parse_operator_metadata/tasks/main.yml

```diff
@@ -104,7 +104,7 @@
     register: kind_custerserviceversion_file_paths
 
   - name: "Grep paths with current_csv ClusterServiceVersion"
-    shell: "grep -l 'name: {{ current_csv }}' {{ item.path }}"
+    shell: "grep -lE \"name: '{{ current_csv }}'|name: {{ current_csv }}\" {{ item.path }}"
```

Comment:
I think the second parameter "name:" should have one space for better readability

Reply:
@sonam1412 
According to grep syntax, the pattern changes if i introduce space and meaning changes. 
I tried it. It does not work as expected.

Reply:
Sure